README.md

```diff
@@ -35,55 +33,63 @@ Backend binaries are output to `/bin`.
 
 ### Configure the application
 
-#### OpenShift
+The following instructions assume you have an existing cluster you can connect
+to. OpenShift 4.0 clusters can be installed using the
+[openshift/installer](https://github.com/openshift/installer). More information
```

Comment:
You can also use CodeReady Containers for local installs. https://github.com/code-ready/osp4